### PR TITLE
Usecase: Set & Get URL search params.

### PR DESCRIPTION
**Pull request by kakadiadarpan**
- [x] Check [CONTRIBUTING.md](https://github.com/jadjoubran/codetogo.io/blob/master/CONTRIBUTING.md)?
- [x] Double check the use case file name?
- [x] Update or remove the `mdn` link?
- [x] Update or remove the `related` use cases?
- [x] Update the category? (if necessary)
- [x] Update the date?

### CODE REVIEW

**Comment by kakadiadarpan** [comment]
Closes #92

**Comment by jadjoubran** [comment]
Thanks @kakadiadarpan! I added some minor comments
Cheers!

File: source/_usecases/set-url-search-params.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+link: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
+related: get-url-search-params
+category: URL
+---
+
+
+```javascript
+const url = new URL('https://codetogo.io');
+
+url.searchParams.set("hello", "world)
```
[the remark this answers is not in the document]
Comment (kakadiadarpan) [reply source/_usecases/set-url-search-params.md]: Good idea! Have done the changes.

File: source/_usecases/set-url-search-params.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+extends: _layouts.usecase
+date: 2018-03-04
+link: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
+related: get-url-search-params
+category: URL
+---
+
+
+```javascript
+const url = new URL('https://codetogo.io');
```
[the remark this answers is not in the document]
Comment (kakadiadarpan) [reply source/_usecases/set-url-search-params.md]: Done.

File: source/_usecases/get-url-search-params.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+extends: _layouts.usecase
+date: 2018-03-04
+link: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
+related: set-url-search-params
+category: URL
+---
+
+
+```javascript
+const url = new URL('https://codetogo.io?hello=world');
```
[the remark this answers is not in the document]
Comment (kakadiadarpan) [reply source/_usecases/get-url-search-params.md]: All changes are done for `get-url..` as well.

**Pull request merged**